pkg/observer/observer.go

```diff
@@ -33,7 +33,7 @@ import (
 )
 
 const (
-	perCPUBufferBytes = 65535
+	perCPUBufferBytes = 1048576
```

Comment:
Good point! The default is zero (I think so that we can distinguish if it was set by the user or not), but we have a comment about what is the default value so I changed that. Thanks!